src/frontend/pages/signup/SignupForm.jsx

```diff
@@ -22,13 +23,13 @@ function SignupForm() {
 				email: email,
 				password: password,
 			});
-
+			console.log(response);
```

Comment:
You can remove this console log from here.

Reply:
thanks for pointing this out